app/views/users/index.html.erb

```diff
@@ -1,3 +1,34 @@
+<div id="merchant-index-statistics">
+  <h1>Top 3 Merchants with the highest profit:</h1>
+  <% User.top_3_merchants_by_profit.each do |merchant| %>
```

Comment:
The only thing that sticks out to me is that calling User in the view is something we are definitely not supposed to do(it isn't following MVC). Is there somewhere in the index that you could do @users instead? It looks like in this case that User is referring to all users so you could just make and instance variable in the index of users and access them that way instead?

Reply:
what do you think about if I put it in the controller, and then it passes the variable @top_merchants_by_profit to the view?

Reply:
I think you're right @SyntheticAutomation, Passing it as an instance variable is the better way.